pallets/randomness/src/lib.rs

```diff
@@ -49,10 +49,12 @@
 //!
 //! `Config::BabeDataGetter` is responsible for reading the epoch index and epoch randomness
 //! from the relay chain state proof. The moonbeam `GetBabeData` implementation is in the runtime.
-
 #![cfg_attr(not(feature = "std"), no_std)]
 
+extern crate alloc;
+
 pub use crate::weights::WeightInfo;
+use alloc::string::String;
```

Comment:
We could also have `Other(Cow<'static, str>)` instead of `Other(String)` to have the possibility to avoid allocation.

Reply:
In fact this error is always used with a static string, so we can avoid 1 allocation

```rust
Ok(Some(InherentError::Other(String::from(
    "Inherent required to set babe randomness results",
))))
```

(why is this not an `InherentError::AlwaysRequired`, I don't know)